pages/applicants.tsx

```diff
@@ -0,0 +1,284 @@
+import type { NextPage } from 'next';
+import styles from '../styles/Applicants.module.scss';
+import Image, { StaticImageData } from 'next/image';
+import { BellIcon, GiftIcon } from '@heroicons/react/24/outline';
+import AdminNavigation from 'components/molecules/Navigation/AdminNavigation';
+
+
+import Button from 'components/atoms/Button/Button';
+import CheckBox from 'components/atoms/CheckBox/CheckBox';
+import DropDown from 'components/atoms/DropDown/DropDown';
+import SearchBar from 'components/atoms/SearchBar/SearchBar';
+
+import ProfileImage from '../assets/applications/ProfilePic.png';
```

Comment:
use [absolute imports](https://nextjs.org/docs/advanced-features/module-path-aliases) instead of relative imports

Example: 
```js
import ProfileImage from 'assets/applications/ProfilePic.png';
```